consumer/dragonboard/qualcomm-robotics-rb5/README.md

```diff
@@ -0,0 +1 @@
+---
title: Using the Qualcomm Robotics RB5 Development kit
permalink: /documentation/consumer/dragonboard/qualcomm-robotics-rb5/
---
# Using the Qualcomm® Robotics RB5 Development Kit 

## Documentation

- [Quick Start Guide](https://developer.qualcomm.com/qualcomm-robotics-rb5-kit/quick-start-guide)
   - Learn about your development board, how to prepare and set up for basic use
- [Software Reference Guide](https://developer.qualcomm.com/qualcomm-robotics-rb5-kit/software-reference-manual)
   - These guides will help to get you started with a variety of available on-boards software functions
- [Hardware Documentation](https://developer.qualcomm.com/qualcomm-robotics-rb5-kit/hardware-reference-guide)
   - Explore what makes your development board unique, technical specifications, schematics, hardware notes and more...

## Software

###With upstream Linux versions by Linaro

A comprehensive guide to using the [Qualcomm® Robotics RB5 Development Kit](https://www.qualcomm.com/products/qualcomm-robotics-rb5-platform) . This guide is written by the [96Boards](https://www.96boards.org) team at [Linaro](http://www.linaro.org) with community contributions and links to third-party content.


- [Downloads](downloads/)
   - Already familiar with your 96Boards? Skip the board bring up sections and go straight to your download!
- [Installation](installation/)
   - Choose and install an operating system on your development kit


### With custom version from Thundercomm

Developers can use the downstream version from Thundercomm to get started. Software and tools to download, build and flash the boot images and rootfs images can be found on Thundercomm's website.

- [Downloads from Thundercomm](https://www.thundercomm.com/app_en/product/1590131656070623)
   - Download the SDK Manager tool from Thundercomm's website to build and flash the images on the development board.
- [Software Tools and Drivers from Qualcomm](https://www.qualcomm.com/products/qualcomm-robotics-rb5-platform/software)
   - Tools and software to support the development platform.


## Support
- [Community Forums](https://developer.qualcomm.com/forums/hardware/robotics/qualcomm-robotics-rb5-kit)
   - From bug reports and current issues, to forum access and other useful resources, we want to help you find answers



```

Comment:
formatting needs to be fixed, weird `^M` characters appearing in-place of  regular `<enter>/<return>`